pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>ru.shemplo.pluses</groupId>
+    <artifactId>pluses</artifactId>
+    <version>1.0</version>
+    <packaging>pom</packaging>
+
+    <name>Pluses Progress Project</name>
+
+    <modules>
+        <module>pluses-message</module>
+        <module>pluses-server</module>
+        <module>pluses-client</module>
+        <module>pluses-echo-server</module>
+    </modules>
```

Comment:
But they are not one project. It's 4 absolutely separate modules that have in common only _Pluses Messages_

Reply:
That is how such project structure should look like. Each module can be used on its own, without parent pom.xml